2-classes-db/task.md

```diff
@@ -56,11 +68,12 @@ To submit this homework write the correct commands after each question.
     the expense area is 'ICT Contingency' (id 18)
     for £32,000.
 ```sql
+insert into spends (expense_type_id, expense_area_id, supplier_id, date, transaction_no, supplier_inv_no,  description, amount) values (16, 18, 16,CAST('2021-04-01' AS date), 38104091, '3780119655', 'Computer Hardware Dell', 32000)
```

Comment:
This is almost perfect, but the initial VALUE should be 7 (the expense_type_id).

Reply:
great, thank you for catching it